kkrt-labs/safe-wallet-web
# Safe wallet

## What it solves
Some spelling mistakes in this file could mislead people who read it, and spelling errors in the code could affect the program file.
Resolves #
Modifications in .github/workflows/deploy-dev.yml to improve the handling of branch names.
Updates in cypress/e2e/happypath/tx_history_filter_hp_1.cy.js to skip flaky tests.
Fix in cypress/e2e/pages/create_tx.pages.js where an incorrect variable name (expande-all-btn) was corrected to expand-all-btn.
## How this PR fixes it
deploy-dev.yml: The branch name extraction logic is updated to ensure the correct format is applied (alphanumeric characters, lowercase, and underscore replacements for non-alphanumeric characters).
tx_history_filter_hp_1.cy.js: Skips the flaky test with a describe.skip() to avoid failures until resolved.
create_tx.pages.js: Corrects a typo where expande-all-btn is changed to expand-all-btn to prevent issues during test execution.
## How to test it
For .github/workflows/deploy-dev.yml, check if the branch name is correctly extracted and formatted in the workflow logs.
For cypress/e2e/happypath/tx_history_filter_hp_1.cy.js, ensure that the flaky test is skipped during test runs.
For cypress/e2e/pages/create_tx.pages.js, verify that the issue with the button ID is resolved and the test runs successfully.
## Screenshots
![срк1](https://github.com/user-attachments/assets/0831c2d4-5061-420c-9641-8622e7a6fe41)
![срк2](https://github.com/user-attachments/assets/c7707df5-e6a3-4126-aeff-9bea67ddca5b)
![срк3](https://github.com/user-attachments/assets/2471b593-fe34-4029-8e98-87c42c04d339)
![срк4](https://github.com/user-attachments/assets/ca2e4bed-9ee1-4a36-9b10-e49d1240ac1e)

## Checklist
